File: ToDoListApp/app/src/main/java/com/example/todolistapp/todoItem.java
package com.example.todolistapp;

public class todoItem {
    private String title;
    private String content;

    public todoItem(String title, String content){
        this.title = title;
        this.content = content;
    }

    public String getTitle(){
        return this.title;
    }

    public String getContent() {
        return this.content;
    }

    public void setContent(String content) {
        this.content = content;
    }

    public void setTitle(String title) {
        this.title = title;
    }
}

File: TodoApplication/app/src/main/java/com/example/todoapplication/TodoItem.kt
package com.example.todoapplication

import java.text.SimpleDateFormat
import java.util.*

class TodoItem {
    var title : String = ""
    var content: String = ""
    var dueTime: Date = Date()
    var complete: Boolean = false

    constructor(title:String, content:String){
        this.title = title;
        this.content = content;
    }



}